editor/modes/visual-editor/style.scss

```diff
@@ -255,3 +255,39 @@ $sticky-bottom-offset: 20px;
 		content: '';
 	}
 }
+
+.editor-visual-editor__placeholder {
```

Comment:
>  I thought we were emulating block effects by duplicating them here.

Yes, this is what we're doing but hard to extract something meaning full without overriding a lot of styles.